schemas/curation.json

```diff
@@ -2,7 +2,10 @@
   "$id": "https://api.clearlydefined.io/schemas/curation.json#",
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "curation",
-  "required": ["coordinates", "revisions"],
+  "required": [
+    "coordinates",
```

Comment:
this reformating is incorrect wrt the project style. Install `prettier` on VS Code and let it format based on the config in the repo.